NaclBox/README.md

```diff
@@ -0,0 +1,103 @@
+# NaclBox Interface
+
+**Status**: Proposed
+
+| Name                | ID                                                                |
+| :------------------ | :---------------------------------------------------------------- |
+| NaclBox	      | da6e573c0282785d9f8e88a81e0f2665811148cb3bd9b5dcce6a079ab7009958  |
+
+
+## Description
+
+NaclBox perform naclbox encryption and decryption. Debot should get the signing box from the SigningBox interface and use it for encryption. For encryption, the browser must deduce curve256 keys from the signing box secret key.
+
+## Functions
+
+Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
+
+`getPublicKey` - get curve256 public key from signing box handle.
```

Comment:
Maybe we are to rename it to make it more clear (to identify this public key is supposed to be used for the encryption)? e.g. `getEncryptPublicKey`